pkg/config/config.go

```diff
@@ -311,6 +311,7 @@ func (c *Config) GetKVParams() kvparams.KV {
 			MaxIdleConnections:    c.values.Database.Postgres.MaxIdleConnections,
 			MaxOpenConnections:    c.values.Database.Postgres.MaxOpenConnections,
 			ConnectionMaxLifetime: c.values.Database.Postgres.ConnectionMaxLifetime,
+			ScanPageSize:          c.values.Database.Postgres.ScanPageSize,
```

Comment:
I don't think this value needs to be configurable. The user had little understanding of what it does, how to configure it and it's much likelier to make a mistake when using it.

Reply:
What I had in mind is us fine tuning this value in production envs, via this configuration. We can leave it configurable and not document it, having it "hiden" from the common user. WDYT?

Reply:
I think this is too internal to be fine-tuned by the user. If anything it's easy to misconfigure this and cause huge latencies. We can always add this later if needed, remove a configuration value is something that is hard to deprecate.

Reply:
Done - removed